### ISSUE TITLE
Some detailed questions

### ISSUE THREAD

**Issue by aaa123-t** [DongqiFu/VCR-Graphormer#2]
Hello, your work is interesting and inspires me a lot！
But recently when running your code in this project, I am a bit confused.
When I directly execute commands you provided in demo.txt, there is a decrease in performance compared to the results in your paper, especially in the PubMed, CoraFull, and Computer datasets.
May I ask if there are some other training tricks, or can you provide commands with better hyperparameters in demo.txt for achieving performance in your paper? Thanks.

**Comment by DongqiFu** [comment]
Hi, thanks very much for your interest!

The commands in the demo.txt can obtain the results as the paper described on our end. If it did not get similar results on your end, I recommend you can change the random seed for the model initialization, i.e., --seed in the command. Thanks.
